frontend/src/components/footer.js

```diff
@@ -0,0 +1,128 @@
+import smartPhoneLogo from "../images/icons/smartphone.svg"
+import facebookLogo from "../images/icons/facebook.svg"
+import instagramLogo from "../images/icons/instagram.svg"
+import twitterLogo from "../images/icons/twitter.svg"
+import copyrightLogo from "../images/icons/Copyright.svg"
+
+const Footer = () => {
+    const browseList = [{title: 'Home', route: '/'}, {title: 'About', route: '/about'}, {title: 'Connect', route: '/'}, {title: 'Next Steps', route: '/'}, {title: 'Events', route: '/'}, {title: 'Watch', route: '/'}, {title: 'Give', route: '/'}];
+    const connectList = [{title: 'Interest Form', route: '/'}, {title: 'Join a LIFE Group', route: '/'}, {title: 'Become a Member', route: '/'}, {title: 'Join a Ministry Team', route: '/'}];
+    return (
+        <div className="bg-Primary-700 justify-center items-center flex">
+            <div className="flex flex-col justify-center items-center max-w-container w-full">
+                <div
+                    className="flex flex-row p-12 w-full justify-between"
+                >
+                    <div className="flex flex-row justify-evenly w-auto gap-[0px] xl:gap-[120px] lg:gap-[100px] md:gap-[50px] sm:gap-[25px] mr-[25px]">
+                        <div className="flex flex-col max-w-[180px] items-start">
+                            <div className="flex flex-col items-center">
+                                <img
+                                alt="hmcc logo"
+                                className="w-[120px] h-[46px]"
+                                src={hmccLogo}
+                                />
+                                <h1 className="text-Shades-0 font-bold text-center text-sm leading-normal -m-2 w-[143px]">HARVEST MISSION COMMUNITY CHURCH</h1>
+                            </div>
+                            <p className="font-medium text-sm leading-[18px] text-Shades-0 mt-[22px]">To transform lost people into Christ’s disciples who will then transform the world.</p>
+                        </div>
+                        <div className="flex flex-col items-start w-fit mt-[30px]">
+                            <div className="items-start flex flex-col">
+                                <h1 className="text-Shades-0 font-bold text-center text-sm leading-normal mb-[12px]">Browse</h1>
+                                {browseList.map(item => (
+                                    <a href={item.route} className="text-Shades-0 font-medium text-start text-sm leading-normal mb-0 no-underline hover:opacity-75">{item.title}</a>
+                                ))}
+                            </div>
+                        </div>
+                        <div className="flex flex-col items-start w-fit mt-[30px]">
+                            <div className="items-start flex flex-col">
+                                <h1 className="text-Shades-0 font-bold text-center text-sm leading-normal mb-[12px]">Ways to Connect</h1>
```

Comment:
there should be only one <h1> tag in the whole page. h# elements is a way to categorize headings for content. for this scenario.  i would suggest to use an <h4> if you using heading. are options are <h1> - <h6>. Could update that too.   looks good overall.